src/aig-common/old-common/services/procedure.resolver.ts

```diff
@@ -0,0 +1,15 @@
+import { Injectable } from '@angular/core';
```

Comment:
PUT THIS FILE IN: /src/aig-common/modules/standard/resolver

Reply:
I did.